src/Symfony/Component/Console/Output/OutputInterface.php

```diff
@@ -91,10 +89,8 @@ public function isDebug();
 
     /**
      * Sets the decorated flag.
-     *
-     * @param bool $decorated Whether to decorate the messages
      */
-    public function setDecorated($decorated);
+    public function setDecorated(bool $decorated);
```

Comment:
We need to update core implementations accordingly, isn't it?

Reply:
yes